master/getting-started/kubernetes/troubleshooting.md

```diff
@@ -53,3 +53,13 @@ However, if you do need to assign a particular address to a Pod, Calico provides
 - You can request an IP using the `cni.projectcalico.org/ipAddrsNoIpam` annotation. Note that this annotation bypasses the configured IPAM plugin, and thus in most cases it is recommended to use the above annotation. 
 
 See the [Requesting a Specific IP address]({{site.baseurl}}/{{page.version}}/reference/cni-plugin/configuration#requesting-a-specific-ip-address) section in the CNI plugin reference documentation for more details.
+
+#### How can I restrict external access to exposed services (i.e. access to a NodePort)?
+
+Limiting access to a NodePort (or any host based service) can be accomplished
```

Comment:
Some comments on this paragraph: the opening sentence is a little difficult. I would suggest simplifying your construction. Adopting the active voice could help a lot. In addition, I am not sure why you have two links here, they both send the reader to the same location. Not just the links, but much of the text seems redundant. I am also not sure why we say "exposed service" in the heading and then switch to "host based service" just below. I think just settling on one or the other and being consistent would be better.

Considering all of the above, I would suggest something as simple as, "Refer to [Host endpoint policy: a worked example]({{site.baseurl}}/{{page.version}}/getting-started/bare-metal/bare-metal#host-endpoint-policy-a-worked-example) for detailed instructions."